docs/topics/proc_configuring-your-application-to-use-vertx.adoc

```diff
@@ -1,16 +1,15 @@
 [id='configuring-your-application-to-use-vertx_{context}']
 = Configuring your application to use {VertX}
 
-To configure your application to use {VertX}:
+Reference the `pom.xml` file in the {VertX} BOM (Bill of Materials) artifact to configure your application.
 
 .Prerequisites
 
 * A Maven-based application
 
 .Procedure
 
-. Reference the {VertX} BOM (Bill of Materials) artifact in the `<dependencyManagement>` section of the `pom.xml` file of your application.
-Specify the `<type>pom</type>` and `<scope>import</scope>`:
+. Add the `<dependencyManagement>` artifact and specify the `<type>pom</type>` and `<scope>import</scope>`:
```

Comment:
It looks like we had a similar conversation before, [see above](https://github.com/fabric8-launcher/launcher-documentation/pull/1239#discussion_r245244400). Also the previous wording

> . Reference the {VertX} BOM (Bill of Materials) artifact in the `<dependencyManagement>` section of the `pom.xml` file of your application.

seems perfectly fine to me, no need to change that. 

The `<dependencyManagement>` section in Maven's pom.xml file is a place where you specify dependency (or artifact) versions and other attributes that are used throughout the project (that is, a place where you centralize dependency information). See [official description.](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management)
 The `vertx-dependencies` Maven artifact (module, ..) is the BOM artifact that does version alignment of other artifacts. For example, it specifies a version of `vertx-web` were you to use it. 

I guess the only thing I would maybe add is a mention about specifying `<scope>` and `<type>`.

Reply:
@mspiniak I am looking to resolve this PR but I want to make sre we are all in agreement here. The text you reference above was fine but its placement within the module was not. This text appeared in the first step of a procedure. All of the steps, which are independent of each other, describe edits to the pom.xml file and the first step is specific to <dependencyManagement>. As a result, I moved the text about the pom.xml to the introductory section of the module. All that said, do you have any comments or want any changes to the module in this PR? Thanks!.

Reply:
@spisiakm ^

Reply:
So I'm still trying to understand why you want to change the wording/steps description. I find [this wording](https://access.redhat.com/documentation/en-us/red_hat_openshift_application_runtimes/1/html-single/eclipse_vert.x_runtime_guide/#configuring-your-application-to-use-vertx_vertx) pretty clear and understandable. The only thing I might do is to remove this sentence

> To configure your application to use Eclipse Vert.x:

from below the heading, as the heading already says that.

Reply:
@spisiakm Sorry about the typo in your handle in my prior post. Thanks to Sheldon for catching and correcting it.

I am not doing a good job of explaining my point so let me try again. The wording you refer to was fine fora simple two-step process. It is bad practice, however, to put information in one step that also applies to other steps in the procedure. The `pom.xml` applies to all steps in this module so I moved that to the top. The `<dependencyManagement>` is specific to step 1 so that remains in step 1. Removing mention `pom.xml` from the first step resulted in awkward working so I recast step 1 to make better sense. Perhaps in vain...:-)

Reply:
@huntley I understand the reasoning, but I think in this case it leads to slightly worse user experience. As a former developer, I tend not to pay a lot of attention to introductions if I know this is indeed the file I'm looking for, and in this case, this means I only glance over it, and go directly to Prerequisites and step 1. That step does not mention where I should add the artifact, so even though I can find that information easily just a few lines above, it makes me stop and think for a second. If you want to adhere to the best practice, I'd say the step 1 should be _Open the `pom.xml` file in the root directory of your application._ or some such to improve the user experience.